pkg/download/downloader.go

```diff
@@ -92,6 +107,38 @@ func extractZIP(targetDir string, read io.ReaderAt, size int64) error {
 	return nil
 }
 
+func isSymlinkAllowed(targetDir, oldname, newpath string) error {
```

Comment:
inconsistent capitalization and wording: targetDir, oldname, newpath.
proposed: targetDir, srcPath, dstPath

Reply:
these inconsistencies come from the surrounding context (os.Symlink). i feel at this point that you should just make the little changes you feel are appropriate?